src/client/login/actions/types.ts

```diff
@@ -9,49 +11,39 @@ export const SET_EMAIL_VALIDATOR = 'SET_EMAIL_VALIDATOR'
 export const IS_LOGGED_IN_SUCCESS = 'IS_LOGGED_IN_SUCCESS'
 export const IS_LOGGED_OUT = 'IS_LOGGED_OUT'
 
-export type GetOtpEmailSuccessAction = {
-  type: typeof GET_OTP_EMAIL_SUCCESS
-  payload: string
-}
+export type GetOtpEmailSuccessAction = ReduxPayloadAction<
+  typeof GET_OTP_EMAIL_SUCCESS,
+  string
+>
+
+export type GetOtpEmailPendingAction = ReduxAction<typeof GET_OTP_EMAIL_PENDING>
```

Comment:
not sure if relevant, but [this](https://stackoverflow.com/questions/44884838/what-is-const-type-pattern-in-typescript) mentions that we can just use `type GET_OTP_EMAIL_PENDING = 'GET_OTP_EMAIL_PENDING'` in line 2 for example and we wont have to use typeof here anymore

Reply:
If the purpose of doing so however is to just avoid `typeof` in the type parameter, that's fairly redundant

Reply:
i think `typeof GET_OTP_EMAIL_PENDING` evaluates to `string`, whereas `type GET_OTP_EMAIL_PENDING = "GET_OTP_EMAIL_PENDING"` is a string that can only be that value. So it depends on how specific we'd like this `ReduxAction<T>` to be